app/code/Magento/Sales/Model/Order.php

```diff
@@ -214,6 +216,11 @@ class Order extends AbstractModel implements EntityInterface, OrderInterface
     protected $_currencyFactory;
 
     /**
+     * @var \Magento\Eav\Model\Config
+     */
+    protected $_eavConfig;
```

Comment:
It was dynamically declared (read: public), making it private may break things in 3rd party code?